openlibrary/templates/subjects.html

```diff
@@ -3,10 +3,23 @@
 $var title: $page.name
 
 $ subject_list = [('subjects', _('Subjects'), 20), ('places', _('Places'), 20), ('people', _('People'), 10), ('times', _('Times'), 10)]
+$ can_add_tag = ctx.user and (ctx.user.is_usergroup_member('/usergroup/super-librarians'))
+$ has_tag = 'tag' in page
 
 <div id="scrollHere"></div>
 
+<!--
+  Able to fetch Tag with the same name as the subject and use it's data to enrich the page via page.tag
+-->
+
 <div class="page-heading-search-box">
+    $if can_add_tag:
+      <div class="subjectTagInfo">
+        $if 'tag' in page:
+            <a href="$page.tag.get('id')/edit" rel="nofollow" title="Edit Subject Tag">$_('Edit Subject Tag')</a>
+        $else:
+            <a href="/tag/add" title="$_('Add a new subject tag')">$_("Add Subject Tag")</a>
```

Comment:
We don't need a separate "Add" link which goes to /tag/add. Instead, let's have the logic just choose / change where the link goes (e.g. either /tag/add or /edit)